DEVELOPMENT.md

```diff
@@ -2,15 +2,41 @@
 
 ## Prerequisites
 
-In order to build Eunomia, you'll need the following:
+Apart from setting up access to a local Kubernetes cluster (described in the [Prerequisites](./GETTING_STARTED.md#Prerequisites) section in the Getting started guide), you will need the following in order to build and test Eunomia:
 
 - [Git](https://git-scm.com/downloads)
 - [Go 1.12+](https://golang.org/dl/)
 - [Docker](https://docs.docker.com/install/)
 - [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
-- Access to a Kubernetes cluster
-  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (optional)
-  - [Minishift](https://www.okd.io/minishift/) (optional)
+- [Minishift](https://github.com/minishift/minishift/releases) (optional; a local OpenShift cluster)
+  - [oc](https://docs.openshift.com/container-platform/3.7/cli_reference/get_started_cli.html#installing-the-cli) (optional; command line tool for controlling OpenShift cluster)
+
+### Installing on a Mac
+
+All the components (except operator-sdk v0.8.1) can be easily installed via [Homebrew](https://brew.sh/) on a Mac using this code snippet:
+
+```shell
+brew install git
+brew install go@1.12
+brew install docker
+brew cask install minishift  # optional
+brew instal openshift-cli  # optional
```

Comment:
I am confused why this list mentions minishift, but doesn't mention minikube; I would expect to see explicit choice between minishift and minikube here.